packages/opine/opine.0.1.3/opam

```diff
@@ -0,0 +1,52 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Python AST unparse implementation in OCaml"
+description:
+  "Opine is an OCaml library that unparses the python AST back to python code."
+maintainer: ["arulselvan1234@gmail.com"]
+authors: ["Arulselvan Madhavan"]
+license: "MIT"
+tags: ["topics" "ocaml" "python" "ast"]
+homepage: "https://github.com/ArulselvanMadhavan/opine"
+doc: "https://url/to/documentation"
+bug-reports: "https://github.com/ArulselvanMadhavan/opine/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "3.6"}
+  "ppx_hash"
+  "ppx_compare"
+  "ppx_sexp_conv"
+  "ppx_make"
+  "ppx_deriving"
+  "pyre-ast" {>= "0.1.6"}
+  "stdio"
+  "base" {>= "0.15.0"}
+  "uutf"
+  "cmdliner"
```

Comment:
```suggestion
  "cmdliner" {>= "1.1.0"}
```
```

#=== ERROR while compiling opine.0.1.3 ========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.14.1 | pinned(https://github.com/ArulselvanMadhavan/opine/archive/refs/tags/0.1.3.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/opine.0.1.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p opine -j 127 @install
# exit-code            1
# env-file             ~/.opam/log/opine-7-a135d4.env
# output-file          ~/.opam/log/opine-7-a135d4.out
### output ###
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/bytecomp/ocaml-compiler-libs.bytecomp.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/common/ocaml-compiler-libs.common.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/shadow/ocaml-compiler-libs.shadow.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/toplevel/ocaml-compiler-libs.toplevel.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlopt.opt -w -40 -g -I bin/.main.eobjs/byte -I bin/.main.eobjs/native -I /home/opam/.opam/4.14/lib/base -I /home/opam/.opam/4.14/lib/base/base_internalhash_types -I /home/opam/.opam/4.14/lib/base/caml -I /home/opam/.opam/4.14/lib/base/shadow_stdlib -I /home/opam/.opam/4.14/lib/cmdliner -I /home/opam/.opam/4.14/lib/parsexp -I /home/opam/.opam/4.14/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.14/lib/ppx_deriving/runtime -I /home/opam/.opam/4.14/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.14/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.14/lib/pyre-ast -I /home/opam/.opam/4.14/lib/pyre-ast/__private__/taglessFinal -I /home/opam/.opam/4.14/lib/result -I /home/opam/.opam/4.14/lib/sexplib -I /home/opam/.opam/4.14/lib/sexplib0 -I /home/opam/.opam/4.14/lib/stdio -I /home/opam/.opam/4.14/lib/uchar -I /home/opam/.opam/4.14/lib/uutf -I lib/.opine.objs/byte -I lib/.opine.objs/native -intf-suffix .ml -no-alias-deps -o bin/.main.eobjs/native/dune__exe__Main.cmx -c -impl bin/main.ml)
# File "bin/main.ml", line 101, characters 13-21:
# 101 |   let info = Cmd.info "expression" ~doc ~man in
#                    ^^^^^^^^
# Error: Unbound module Cmd
```